src/components/teamMember.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react"
+import "./teamMember.scss"
+export default function TeamMember({ member }) {
+  const { name, role, descriptions, imageSrc } = member
+  const [expanded, setExpanded] = useState(false)
+  const collapsedClasses =
+    "bio-collapsible__collapsible-content bio-collapsible__collapsible-content--collapsed"
+  const expandedClasses =
+    "bio-collapsible__collapsible-content bio-collapsible__collapsible-content--expanded"
```

Comment:
Not your fault, but these class names are killing me :open_mouth:

Reply:
My apologies, I agree those are some nasty classes haha. I shortened them up.